### PR TITLE
Os install reqs

### PR DESCRIPTION
**Pull request by avanbrunt-cb**
## Pull request checklist

Please check if your PR fulfills the following requirements:

- [x] Docs have been reviewed and added / updated if needed (for bug fixes / features)
- [ ] Tests have been added that prove the fix is effective or that the feature works.
- [ ] New and existing tests pass locally with the changes.
- [ ] Code follows the style guidelines of this project (PEP8, clean code).
- [ ] Linter has passed locally and any fixes were made for failures.
- [ ] A self-review of the code has been done.

## Pull request type

Please check the type of change your PR introduces:
- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [x] Documentation content changes (not tied to bugs/features)
- [ ] Other (please describe):

## What is the ticket or issue number?

CBAPI-1569

## Pull Request Description

Add requirements for compiling yara-python on Ubuntu and Windows
Make python 3.6 supported

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

## How Has This Been Tested?

Documentation only but verified with manual review

### CODE REVIEW

**Comment by bvasil-cb** [comment]
Yep looks reasonable.

File: README.md [hunk 1]
```diff
@@ -41,6 +41,16 @@ All requirements are installed as part of `pip install` or if you're planning on
 * schema
 * yara-python
 
+**Note: Windows users will need to have [Microsoft Visual C++ 14.0 Build Tools](https://visualstudio.microsoft.com/visual-cpp-build-tools) installed in order to compile yara-python.**
+
+**Note: Linux users will need to have the python developer package installed in order to compile yara-python. If there are compile errors make sure to be on the latest gcc compiler**
```
Comment (lhilmes-cb) [inline README.md:46]: Consider changing second sentence to "If you receive compile errors, make sure you are on the latest..."

**Comment by kebringer-cb** [comment]
Version and release number need updating, since there's a code change.

File: README.md [hunk 2]
```diff
@@ -41,6 +41,16 @@ All requirements are installed as part of `pip install` or if you're planning on
 * schema
 * yara-python
 
+**Note: Windows users will need to have [Microsoft Visual C++ 14.0 Build Tools](https://visualstudio.microsoft.com/visual-cpp-build-tools) installed in order to compile yara-python.**
```
Comment (lhilmes-cb) [inline README.md:44]: Consider having one "Note" with two bullet points below, one for Windows and one for Linux (the concern is too many notes can create note-blindness)

**Note:**
- **Windows** users will need to have...
- **Linux** users will need to have...

- 

- 

-

**Comment by avanbrunt-cb** [comment]
@kebringer-cb I wasn't planning on updating that with this PR as we have Amy's PR and wanted to let the last bit of acceptance testing finish to see if there are other quick bug fixes

**Comment by abowersox-cb** [comment]
I'd call it 1.0a2 in that case.

**Comment by kebringer-cb** [comment]
Version will be bumped prior to release.  Other changes are good to go.

**Pull request merged**